code/services/QueuedJobService.php

```diff
@@ -325,6 +325,9 @@ public function checkJobHealth($queue = null) {
 	 */
 	public function checkdefaultJobs($queue = null) {
 		$queue = $queue ?: QueuedJob::QUEUED;
+
+		$this->defaultJobs = !empty($this->defaultJobs) ? $this->defaultJobs : Config::inst()->get(__CLASS__, 'defaultJobs');
```

Comment:
Missing the `properties` level nest there:
```
Injector:
  QueuedJobService:
    properties:
      defaultJobs: 
```

I missed that while testing so had had the config sitting outside the Injector config array. I've updated the readme to add it and remove the unnecessary check.